plugins/coverbrowser.koplugin/listmenu.lua

```diff
@@ -541,10 +541,12 @@ function ListMenuItem:update()
 end
 
 function ListMenuItem:paintTo(bb, x, y)
-    -- We may get non-integer x or y (see mosaicmenu.lua)
-    -- Make them integer:
-    x = math.floor(x)
-    y = math.floor(y)
+    -- We used to get non-integer x or y that would cause some mess with image
+    -- inside FrameContainer were image would be drawn on top of the top border...
+    -- Fixed by having TextWidget:updateSize() math.ceil()'ing its length and height
+    -- But let us know if that happen again
+    assert(x == math.floor(x))
```

Comment:
I hardly never run in debug mode on my kobo, only on the emulator.
And this drawing mess happened only on kobo and not on emulator (i run the emulator as a 600x610 window on a small laptop screen - and we had only non-integer x/y with window size > 1000x1000) - so I wouldn't have catch it with Dbg:assert().
I admit that these assert() are a bit radical (even if we shouldn't have non-integer pixel anymore).
I would be fine using logger.warn() if there x ~= math.floor(x). Should i do that instead ?
Or is there a logging only equivalent of assert ?